clairmeta/dcp_check_subtitle.py

```diff
@@ -721,3 +720,34 @@ def check_subtitle_cpl_image(self, playlist, asset, folder):
                 raise CheckException(
                     "Subtitle image reference {} not found in folder {}"
                     "".format(img, os.path.relpath(folder, self.dcp.path)))
+
+    def check_subtitle_cpl_first_tt_event(self, playlist, asset, folder):
+        """ First TT Event of Composition check
+
+            The composition's first Timed Text event's TimeIn attribute
+            should be greater than or equal to 4 seconds.
+
+            Reference:
+                SMPTE RDD 52:2020 7.2.4
+
+        """
+
+        reel_cpl = get_reel_for_asset(playlist, asset[1]['Id'])['Position']
+        first_reel_of_st = get_first_reel_for_asset_type(playlist, 'Subtitle')
```

Comment:
See comment about returning the reel and querying `['Position']` below.

Reply:
See my comment above.